docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
   * refactor
   * test (when adding missing tests)
   * chore (maintain)
-* Scope: anything that specifies the scope of the commit. Can be blank or `*`
-* Subject: description of the commit. For **breaking changes** that require major version bump, add `BREAKING CHANGE` to the commit message.
+* Scope: anything that specifies the scope of the commit; can be blank, the issue number that your commit pertains to, or `*`
+* Subject: description of the commit
 
-**Examples commit messages:**
-* Bug fix: `fix: Remove extra space`
-* Breaking change: `feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`
+**Important:** For any **breaking changes** that require a major version bump, add `BREAKING CHANGE` somewhere in the commit title or message.
+
+**Examples commit titles:**
+* For a bug fix: `fix: Remove extra space`
+* For a breaking change: `feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`
 
 ## Feature Requests
 
-Make the case for a feature via an issue with a good title. The feature should be discussed and given a target inclusion milestone or closed.
+Make the case for a feature via an issue with a good title. The feature will then be discussed and either given a target inclusion milestone or closed.
 
 ## Where to Contribute
 
-Screwdriver has a modular architecture, and the various responsibilities are split up in separate repos.
+Screwdriver has a modular architecture, and the various responsibilities are split up into separate repos.
+
+Check out the [architecture diagram](arch-diagram) to see the overall workflow of Continuous Delivery using Screwdriver. The next few sections will help lay out where different code repositories fit.
 
-### [Screwdriver API][api-repo] [![Version][api-npm-image]][api-npm-url] [![Open Issues][api-issues-image]][api-issues-url]
+### [Screwdriver API][api-repo]
 The **[screwdriver][api-repo]** repo is the core of screwdriver, providing the API endpoints for everything that screwdriver does. The API is based on the *[hapijs framework](http://hapijs.com/)* and is implemented in node as a series of plugins.
 
+
+* **[Build bookends][build-bookend-repo]** allow a user to create setup and teardown steps for builds.
```

Comment:
Hm I figured bookends and notifications were just smaller things generally related to the API. It seemed excessive to have them as their own sections. If you feel strongly about separating them out, I can do that.